swim/ping_request_sender.go

```diff
@@ -112,6 +115,29 @@ func (p *pingRequestSender) MakeCall(ctx json.Context, res *pingResponse) <-chan
 	return errC
 }
 
+// indirectPing is used to check if a target node can be reached indirectly.
+// The indirectPing is performed by sending a specifiable amount of ping
+// requests nodes in n's membership.
+func indirectPing(n *Node, target string, amount int, timeout time.Duration) (reached bool, errs []error) {
```

Comment:
Only reason I suggested removing them is because of the way I've seen Go's named return values previously used. They're used as a replacement actually returning values in various parts of a method and not to serve as documentation.